nginx_conf.sh

```diff
@@ -1,28 +1,34 @@
 #!/bin/bash
 
-# Updating and installing required commands
-apt update  -y
+# installing required commands
 apt install wget -y
 apt install curl -y
```

Comment:
before installing check if *wget* and *curl* are already installed

Reply:
done